lib/atmosphere.rb

```diff
@@ -48,7 +48,7 @@ def self.admin_entities
   # manage.
   mattr_accessor :at_pdp_class
   def self.at_pdp(user)
-    (at_pdp_class || Atmosphere::DefaultPdp).new(user)
+    at_pdp_class.new(user)
```

Comment:
To ensure that the `DefaultPDP` is not used when the request specifies otherwise (for example, when the PDP cannot be determined by analyzing the request). `DefaultPDP` is permissive, it would be a security risk to keep it as 'default' - safer to generate an exception.